src/main/java/com/sigmaukraine/messenger/utils/Encryption.java

```diff
@@ -0,0 +1,17 @@
+package com.sigmaukraine.messenger.utils;
+
+
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class Encryption {
+
+    public String md5Encode(String string) throws NoSuchAlgorithmException {
+
+        MessageDigest digest = MessageDigest.getInstance("MD5");
```

Comment:
As I mentioned earlier, it would be cool if you switch to bcrypt. Also spring-security already contains it's implementation and you can `@Autowire` it and call whenever you want.